sbat.c

```diff
@@ -335,4 +335,62 @@ parse_sbat_var(list_t *entries)
 	FreePool(data);
 	return EFI_INVALID_PARAMETER;
 }
+
+EFI_STATUS
+init_sbat()
```

Comment:
same here.

Reply:
Can we possibly call this something else?  We now have *three* things we initialize that are effectively called "sbat":
- the `SBAT` variable in EFI
- our `SBAT` variable data: `sbat_var` nearly everywhere now
- the `.sbat` section in a binary we're loading: `sbat_section` nearly everywhere now
Maybe `set_sbat_variable()`?

Reply:
Yes, how about: set_sbat_uefi_variable() ?